plugins/slack/app/views/samson_slack/_fields.html.erb

```diff
@@ -1,18 +1,22 @@
 <fieldset>
   <legend>Slack</legend>
-  <p class="col-lg-offset-2">Name of the slack channel which should be notified of deploys.</p>
+  <p class="col-lg-offset-2">Details of the webhook to notify on deploys.</p>
 
   <% stage = form.object %>
-  <% stage.slack_channels.build if stage.slack_channels.last.try(:name).blank? %>
+  <% stage.slack_webhooks.build if stage.slack_webhooks.last.try(:webhook_url).blank? %>
 
-  <%= form.fields_for :slack_channels do |slack_fields| %>
+  <%= form.fields_for :slack_webhooks do |slack_fields| %>
     <div class="form-group">
-      <div class="col-lg-2 col-lg-offset-2">
-        <%= slack_fields.text_field :name, class: "form-control", placeholder: "Channel name" %>
+      <div class="col-lg-6 col-lg-offset-2">
+        <%= slack_fields.text_field :webhook_url, class: "form-control", placeholder: "Webhook URL" %>
+      </div>
+
+      <div class="col-lg-3 col-lg-offset-2">
+        <%= slack_fields.text_field :channel, class: "form-control", placeholder: "Channel (optional)" %>
       </div>
 
       <% if slack_fields.object.persisted? %>
-        <div class="col-lg-1 checkbox">
+        <div class="col-lg-offset-2 col-lg-10">
```

Comment:
I'm being super nitpicky, but can we add the paragraph descriptor inside the form-group and have all three divs on the same line?

```
<p class="col-lg-8 col-lg-offset-2"></p>
<div class="col-lg-4 col-lg-offset-2"></div>
<div class="col-lg-3"></div>
<div class="col-lg-1 checkbox"></div>
```

And for this last one, place the actual checkbox inside the label:
```
<%= slack_fields.label ... do %>
  <%= slack_fields.check_box :_destroy %>
  Destroy
<% end %>
```

It'll end up looking like this:
![screen shot 2015-11-30 at 10 27 36 am](https://cloud.githubusercontent.com/assets/13248/11480577/05d7a7b4-974d-11e5-8117-315e4c6b9768.png)